mobile/tradeverse/screens/portfolio/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { View, Text, ScrollView, StyleSheet, TouchableOpacity } from 'react-native';
+import GlobalScreen from "../../components/ui/global-screen"
+import FullScrollView from "../../components/ui/full-scroll-view"
+// Mock Data
+const portfolioData = [
+  { name: 'BTC', value: '0.005' },
+  { name: 'USD', value: '$31.82' },
+  { name: 'EUR', value: '€52.96' },
+  { name: 'JPY', value: '¥4000.50' },
+  { name: 'BTC', value: '0.005' },
+  { name: 'USD', value: '$31.82' },
+  { name: 'EUR', value: '€52.96' },
+  { name: 'JPY', value: '¥4000.50' },
+  { name: 'BTC', value: '0.005' },
+  { name: 'USD', value: '$31.82' },
+  { name: 'EUR', value: '€52.96' },
+  { name: 'JPY', value: '¥4000.50' },
+  { name: 'BTC', value: '0.005' },
+  { name: 'USD', value: '$31.82' },
+  { name: 'EUR', value: '€52.96' },
+  { name: 'JPY', value: '¥4000.50' },
+  // You can add more assets as needed
+];
+
+
+const PortfolioScreen = () => {
+  return (
+  <GlobalScreen>
+  <FullScrollView>
+    <View style={styles.container}>
+
+      <View style={styles.titleBlock}>
+        <Text style={styles.title}>My Portfolio</Text>
+        <TouchableOpacity style={styles.addButton}>
+          <Text style={styles.addButtonText}>+</Text>
+        </TouchableOpacity>
+      </View>
+
+      {/* Scrollable Portfolio Block */}
+      <ScrollView contentContainerStyle={styles.scrollViewContent}>
+        <View style={styles.portfolioBlock}>
+          {portfolioData.map((asset, index) => (
+            <TouchableOpacity key={index} style={styles.assetBlock}>
+              <Text style={styles.assetName}>{asset.name}</Text>
+              <Text style={styles.assetValue}>{asset.value}</Text>
+            </TouchableOpacity>
+          ))}
+        </View>
+      </ScrollView>
+    </View>
+    </GlobalScreen>
+    </FullScrollView>
+  );
+};
+
+// Styles
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#EDE7F6', // Light purple background
+    padding: 20,
```

Comment:
We do not need this padding since we already have it on GlobalScreen

Reply:
I have removed that padding as well